### DISCUSSION TITLE
phalconv5 on docker

### DISCUSSION THREAD

**Discussion by davidcorreia** [phalcon/cphalcon#15979] [General]
Hey awsome news!
version 5 is here :) https://blog.phalcon.io/post/phalcon-v5-0-0rc1-released

how ever i'm having some problems on getting it to work on docker with the Dockerfile from repo https://github.com/phalcon/cphalcon/tree/master/docker/8.1

After the build
> $ docker run 2661df2671cc -i|grep phalcon -A 5
> MY_USER => phalcon
> MY_GROUP => phalcon
> $_SERVER['MY_USER'] => phalcon
> $_SERVER['MY_GROUP'] => phalcon
> $_ENV['MY_USER'] => phalcon
> $_ENV['MY_GROUP'] => phalcon

was expection to see the version 
> $ php -i|grep phalcon -A 5|grep Version
> Version => 4.1.2

am i doing something wrong?

**Comment by Jeckerson** [comment]
> https://github.com/phalcon/cphalcon/tree/master/docker/8.1

This `Dockerfile` designed specifically for local environment, please don't use them.

If you need working example of production ready `Dockerfile`, please see:
https://github.com/MilesChou/docker-phalcon/pull/50/files#diff-f12575bad558ce63d4af0954d08b6016d906c38fb1624dfaf9aa75963dbf1b54R1

Also note, that `psr` extension since version `phalcon-RC1` is not required.
